redaxo/src/core/lib/sql/table.php

```diff
@@ -918,13 +919,19 @@ private function getColumnDefinition(rex_sql_column $column)
             $default = 'DEFAULT '.$this->sql->escape($column->getDefault());
         }
 
+        $comment = $column->getComment();
+        if ($comment) {
```

Comment:
```suggestion
        if (null !== $comment && '' !== $comment) {
```

Zwar unwahrscheinlich, dass es vorkommt, aber so unterstützen wir auch Kommentare mit dem Text "0".